### ISSUE TITLE
maybe a shape bug in create_dataset_from_pano.py? 

### ISSUE THREAD

**Issue by JizheXIA** [cvg/GeoCalib#10]
in https://github.com/cvg/GeoCalib/blob/main/siclib/geometry/base_camera.py#L471:
pano_shape = (int(pano_img.shape[0] * scale), int(pano_img.shape[1] * scale))

but pano_img is a Bx3xHxW tensor, maybe there should be "pano_img[2]" and "pano_img[3]" instead of "pano_img[0]" and "pano_img[1]"?

**Comment by sarlinpe** [comment]
Good catch, thank you! @veichta knows more about this part of the code.

**Comment by veichta** [comment]
Thank you for pointing this out! It should be fixed.
